spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-polling-sample/src/main/java/com/example/PollableSink.java

```diff
@@ -1,31 +0,0 @@
-/*
```

Comment:
Yes, we need to use Spring Cloud Function since the legacy imperative model is removed in Spring Cloud 4.0.

Spring Cloud 4.0 it's not GA yet so I can't refer the Migration guide to you but you can read this [Spring blog](https://spring.io/blog/2019/10/17/spring-cloud-stream-functional-and-reactive) for more info.